ci-operator/config/openshift/lightspeed-operator/openshift-lightspeed-operator-main.yaml

```diff
@@ -107,7 +114,53 @@ tests:
       resources:
         requests:
           cpu: 100m
-    workflow: hypershift-hostedcluster-workflow
+    workflow: generic-claim
```

Comment:
We move away from hypershift clusters to support different versions ?

Reply:
@raptorsun for the time being. We need to start a thread with testplatform team to find out how we can have multiple hypershift versions pointing to the same development branch (because the hypershift version sets it for the whole file and we can't have different files for the same development branch)